### PR TITLE
feat: migration some the designer image

### PR DESCRIPTION
**Pull request by RealKai42**
Upload some designer image which are already exists in MCR

### CODE REVIEW

File: assets/designer/environments/designer-cv-transform/context/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM mcr.microsoft.com/azureml/openmpi4.1.0-ubuntu20.04:latest
```
Comment (vizhur) [inline assets/designer/environments/designer-cv-transform/context/Dockerfile:1]: use template to resolve immutable date based tag during the build, see other examples in the repo

File: assets/designer/environments/designer-cv-transform/context/conda_dependencies.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+name: project_environment
+channels:
+  - defaults
```
Comment (vizhur) [inline assets/designer/environments/designer-cv-transform/context/conda_dependencies.yaml:3]: I'd recommend to put conda-forge channel first. Channel is well maintained, defaults (anaconda) is usually significantly out of date

File: assets/designer/environments/designer-cv-transform/context/conda_dependencies.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  - pytorch
+dependencies:
+  - cpuonly
+  - pip=20.2
+  - python=3.8.10
+  - pytorch=2.3.1
+  - torchvision=0.18.1
+  - pip:
+      - azureml-designer-cv-modules==0.0.59
+      - setuptools==65.5.1
```
Comment (vizhur) [inline assets/designer/environments/designer-cv-transform/context/conda_dependencies.yaml:13]: I don't think you need packages like wheel, setuptools explicitly referenced here. I believe that was done to mitigate vulnerabilities in the past, but with prioritized conda-forge channel, usually you will get the latest of those every time you materialize conda environment

File: assets/designer/environments/designer-cv/environment.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  os: linux
+  context:
+    dir: context
+    dockerfile: Dockerfile
+    template_files:
+      - Dockerfile
+      - conda_dependencies.yaml
+  publish:
+    location: mcr
+    visibility: unlisted
```
Comment (vizhur) [inline assets/designer/environments/designer-cv/environment.yaml:12]: visibility must be public

File: assets/designer/environments/designer-cv-transform/asset.yaml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+name: designer-cv-transform
+version: auto
```
[the remark this answers is not in the document]
Comment (RealKai42) [reply assets/designer/environments/designer-cv-transform/asset.yaml:2]: It seems like I cannot get the old image info like designer-cv-transform in _azureml_ registry

**Pull request merged**